cel/config/500-controller.yaml

```diff
@@ -15,52 +15,49 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: cel-controller
-  namespace: tekton-cel-run
+  name: tekton-cel-controller
+  namespace: tekton-pipelines
   labels:
-    app.kubernetes.io/name: cel-controller
-    app.kubernetes.io/component: cel-controller
+    app.kubernetes.io/name: controller
+    app.kubernetes.io/component: controller
     app.kubernetes.io/instance: default
-    app.kubernetes.io/version: devel
-    app.kubernetes.io/part-of: tekton-cel-run
+    app.kubernetes.io/part-of: tekton-cel
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app.kubernetes.io/name: cel-controller
-      app.kubernetes.io/component: cel-controller
+      app.kubernetes.io/name: controller
+      app.kubernetes.io/component: controller
       app.kubernetes.io/instance: default
-      app.kubernetes.io/part-of: cel
+      app.kubernetes.io/part-of: tekton-cel
   template:
     metadata:
       annotations:
         cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
       labels:
-        app.kubernetes.io/name: cel-controller
-        app.kubernetes.io/component: cel-controller
+        app.kubernetes.io/name: controller
+        app.kubernetes.io/component: controller
         app.kubernetes.io/instance: default
-        app.kubernetes.io/version: devel
-        app.kubernetes.io/part-of: cel
-        app: cel-controller
+        app.kubernetes.io/part-of: tekton-cel
+        app: tekton-cel-controller
     spec:
-      serviceAccountName: cel-controller
+      serviceAccountName: tekton-cel-controller
       containers:
-        - name: cel-controller
-          image: ko://github.com/tektoncd/experimental/cel/cmd/controller
-          volumeMounts:
-            - name: config-logging
-              mountPath: /etc/config-logging
-          env:
-            - name: SYSTEM_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
-            # If you are changing these names, you will also need to update
-            # the controller's Role in 200-role.yaml to include the new
-            # values in the "configmaps" "get" rule.
-            - name: CONFIG_LOGGING_NAME
-              value: config-logging
-      volumes:
-        - name: config-logging
-          configMap:
-            name: config-logging
+      - name: tekton-cel-controller
+        image: ko://github.com/tektoncd/experimental/cel/cmd/controller
+        env:
+        - name: SYSTEM_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: CONFIG_LEADERELECTION_NAME
+          value: config-leader-election
+        - name: CONFIG_LOGGING_NAME
+          value: config-logging
+        - name: CONFIG_OBSERVABILITY_NAME
+          value: config-observability
+        - name: METRICS_DOMAIN
+          value: tekton.dev/pipeline
```

Comment:
Probably want to change this - I assume this is some sort of bucketing mechanism so we want to make sure we're not group metrics alongside the main pipeline controller.